VMEncryption/main/patch/redhatPatching.py

```diff
@@ -324,6 +324,8 @@ def install_and_enable_ade_online_enc(self, root_partuuid, boot_uuid, rootfs_dis
         # Change config so that dracut will force add the dm_crypt kernel module
         self.append_contents_to_file('\nadd_drivers+=" dm_crypt "\n',
                                       '/etc/dracut.conf.d/ade.conf')
+        # Change config so that dracut will force add the cryptsetup binary
```

Comment:
Is it still needed after Azure Linux 3 fix? 
If needed, then probably should only be added for azure linux 3.

Reply:
I think we should rely on the system to have the necessary binary included in the initrd, as we were doing for redhat. 
If for some reason azure linux 3 is still not working then this change should be isolated to azure linux 3.
As you mentioned, it might cause some unknown conflicts and that would be difficult to debug.